configure

```diff
@@ -368,7 +368,7 @@ fi
 # Generate ortho_code-x86-flags
 if test $backend = mcode; then
   case "$build" in
-    *darwin*) ortho_flags="Flags_Macosx${mcode64}" ;;
+    *darwin* | *freebsd* ) ortho_flags="Flags_Macosx${mcode64}" ;;
```

Comment:
Is there any reason to use Flags_Macosx instead of Flags_Linux ?
The difference is the stack alignment on x86.  For darwin it is 16 bytes, for linux it is 8 bytes.
I thought that Freebsd is closer to linux.